src/main/java/ti4/service/strategycard/PlayStrategyCardService.java

```diff
@@ -397,6 +398,24 @@ private static void sendAndHandleMessageResponse(MessageCreateData toSend, Game
             threadChannel.queue(m5 -> {
                 if (game.getOutputVerbosity().equals(Constants.VERBOSITY_VERBOSE) && scModel.hasImageFile()) {
                     MessageHelper.sendMessageToChannel(m5, Helper.getScImageUrl(scToPlay, game));
+                    if (ShouldPrintFollowOrder(game, scModel)) {
+                        List<Player> playersInOrder = getPlayersInFollowOrder(game, player);
+                        StringBuilder playerOrder = new StringBuilder("__Order for performing the Secondary ability:__\n");
+                        for (int i = 0; i < playersInOrder.size(); i++) {
+                            playerOrder.append("`").append(i + 1).append(".` ");
+                            if (game.isOmegaPhaseMode() && game.getPhaseOfGame().equals("action")) {
+                                int lowestSC = playersInOrder.get(i).getLowestSC();
+                                TI4Emoji scEmoji = CardEmojis.getSCFrontFromInteger(lowestSC);
+                                playerOrder.append(scEmoji);
+                            }
+                            playerOrder.append(playersInOrder.get(i).getRepresentationNoPing());
+                            if (playersInOrder.get(i).isSpeaker()) {
+                                playerOrder.append(MiscEmojis.SpeakerToken);
+                            }
+                            playerOrder.append("\n");
+                        }
+                        MessageHelper.sendMessageToChannel(m5, playerOrder.toString());
+                    }
```

Comment:
Omega Phase has a weird order for performing the Secondary Ability of strategy cards: Initiative order. This is because it completely removes Speaker order, and the thing it replaces it with (Priority Track) isn't available in the Action phase.

This change will helpfully print the secondary order as a reminder to folks, and also prints it in standard games with Research Agreement so folks can see whether or not JolNar techs can be used as pre-reqs.